README.md

```diff
@@ -75,3 +75,10 @@ SELECT month, yoy_growth
 FROM repo_commits_growth
 WHERE repo_name = "plotly/plotly.py"
 ```
+
+## Pipeline Directed Acyclic Graph (DAG)
```

Comment:
Style: I'd suggest putting the images closer to the top of the readme (visuals are good!). Move these 2 new sections to the top.

Reply:
For this section, a better title could be "Lineage Graph".